enclave-tls/src/enclave_quotes/sgx-la/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <enclave-tls/enclave_quote.h>
+#include <enclave-tls/log.h>
+
+extern enclave_quote_err_t enclave_quote_register(enclave_quote_opts_t *);
+extern enclave_quote_err_t sgx_la_pre_init(void);
+extern enclave_quote_err_t sgx_la_init(enclave_quote_ctx_t *,
+				     enclave_tls_cert_algo_t algo);
+extern enclave_quote_err_t sgx_la_collect_evidence(enclave_quote_ctx_t *,
+						 attestation_evidence_t *,
+						 enclave_tls_cert_algo_t algo,
+						 uint8_t *);
+extern enclave_quote_err_t sgx_la_verify_evidence(enclave_quote_ctx_t *,
+						attestation_evidence_t *, uint8_t *);
+extern enclave_quote_err_t sgx_la_cleanup(enclave_quote_ctx_t *);
+
+static enclave_quote_opts_t sgx_la_opts = {
+	.api_version = ENCLAVE_QUOTE_API_VERSION_DEFAULT,
+	.flags = ENCLAVE_QUOTE_FLAGS_DEFAULT,
+	.type = "sgx_la",
+	.priority = 53,
```

Comment:
Plz change to 54.

50: sgx epid
51: tdx ecdsa
52: sgx ecdsa
53: tdx la
54: sgx la